src/Distributed/Distributed.jl

```diff
@@ -19,5 +18,6 @@ include("halo_communication_bcs.jl")
 include("distributed_fields.jl")
 include("halo_communication.jl")
 include("distributed_fft_based_poisson_solver.jl")
+include("interleave_comm_and_comp.jl")
```

Comment:
```suggestion
include("interleave_communication_and_computation.jl")
```

Please don't use abbreviations.